docs/zh-CN/README.md

```diff
@@ -194,94 +193,91 @@ ThirdParty
+[此页面](https://overlayplugin.github.io/cactbot/util/coverage/coverage.html) 列出了当前 Cactbot 中支持的副本。我们会持续添加更多支持 (随时欢迎贡献代码！) 尽管许多旧副本仍未支持。
 
-战斗时间轴原本是设计为用于 [ACT 时间轴](https://github.com/grindingcoil/act_timeline)插件的文件，不过增加了一些[扩展语法](./TimelineGuide.md)。这里是原插件的 [文档](http://dtguilds.enjin.com/forum/m/37032836/viewthread/26353492-act-timeline-plugin)。
+战斗时间轴原本是设计为用于 [ACT 时间轴](https://github.com/grindingcoil/act_timeline)插件的文件，不过增加了一些[扩展语法](./TimelineGuide.md)。这里是原插件的 [文档](https://web.archive.org/web/20230426121530/https://dtguilds.enjin.com/forum/m/37032836/viewthread/26353492-act-timeline-plugin)。
 
 这里有三个等级的警报提示，重要性从低到高分别为：`info(信息)`、`alert(警告)` 和 `alarm(警报)`。文本信息只会有这三种等级，等级越高，越重要，则文字会越大，颜色也会越醒目。如果你更喜欢文本到语音 (TTS)，你也可以配置成语音提示。
 
-在[ui/raidboss/data](../../ui/raidboss/data)文件夹下存放了定义了文本显示和声音提示的时间轴和触发器等， 时间轴文件拥有 `.txt` 扩展名，而触发器文件则为 `.ts` 扩展名。
+在 [ui/raidboss/data](../../ui/raidboss/data) 文件夹下存放了定义了文本显示和声音提示的时间轴和触发器等，时间轴文件拥有 `.txt` 扩展名，而触发器文件则为 `.ts` 扩展名。
 
-在下方的截图中，高亮的是 raidboss 模块，其中时间轴用红色圆圈圈出，文本警报用黄色圆圈圈出，可见的是 `警告`等级的文字提示。
+在下方的截图中，高亮的是 Raidboss 模块，其中时间轴用红色圆圈圈出，文本警报用黄色圆圈圈出，可见的是 `警告` 等级的文字提示。
 
 ![raidboss 屏幕截图](../../screenshots/Raidboss.png)
 
-### raidboss 模拟器
+### Raidboss 模拟器
 
-如果在编写触发器或时间轴，并要对其进行测试，您可以使用此处的 raidboss 模拟器：**ui/raidboss/raidemulator.html**。
+如果在编写触发器或时间轴，并要对其进行测试，你可以使用此处的 Raidboss 模拟器：**ui/raidboss/raidemulator.html**。
 
-但是，目前只能在浏览器中加载它，不支持加载为悬浮窗。 该模拟器在最新版本的 Chrome 中可以完美运行， 理论上在其他浏览器中也可以运行，但是并没有测试过。
+但是，目前只能在浏览器中加载它，不支持加载为悬浮窗。该模拟器在最新版本的 Chrome 中可以完美运行，理论上在其他浏览器中也可以运行，但是并没有测试过。
 
-操作步骤：
+如果你想让模拟器使用 ACT 的设置和用户触发器，你可以通过以下操作步骤启用 OverlayPlugin WS 服务器：
 
 1. 启动 ACT。
-1. 确保 WS Server 已启动，可以在 Plugins -> OverlayPlugin WSServer-> Stream/Local Overlay 中对此进行配置。
+1. 确保 WS Server 已启动，可以在 Plugins -> OverlayPlugin WSServer -> Stream/Local Overlay 中对此进行配置。
 
-如果您正在为cactbot仓库开发触发器，
-您可以通过 `npm run start` 启动本地开发服务器，
-并在Chrome中访问 `http://127.0.0.1:8080/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws`
+如果你正在为 Cactbot 仓库开发触发器，你可以通过 `npm run start` 启动本地开发服务器，并在Chrome中访问 `http://127.0.0.1:8080/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws`。
 
-如果您正在开发用户自定义触发器，
-您可以在Chrome中访问 `https://overlayplugin.github.io/cactbot/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws`
+如果你正在开发用户自定义触发器，你可以在 Chrome 中访问 `https://overlayplugin.github.io/cactbot/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws`。
 
-如果您正在尝试复现一个问题，
-您可以在Chrome中访问 `https://overlayplugin.github.io/cactbot/ui/raidboss/raidemulator.html`
-在这种情况下，您不需要运行WS服务器。
+如果你正在尝试复现一个问题，你可以在 Chrome 中访问 `https://overlayplugin.github.io/cactbot/ui/raidboss/raidemulator.html`。在这种情况下，你不需要运行WS服务器。
 
 成功加载网页后，继续按照以下说明使用模拟器。
 
-1. 拖放一个 [网络日志](/docs/FAQ-Troubleshooting.md#how-to-find-a-network-log) 文件到该页面中。
+1. 拖放一个 [网络日志](../FAQ-Troubleshooting.md#how-to-find-a-network-log) 文件到该页面中。
 1. 选择区域和战斗记录，然后单击 `Load Encounter`。
 
 如果模拟器无法正常工作，请检查控制台中是否显示了错误日志。
 
 ![Raidboss模拟器截图](/screenshots/raidboss_emulator.png)
```

Comment:
The `screenshots` elsewhere are relative paths, but only this one is absolute.

Reply:
I changed it to relative path now, thanks for pointing out.